src/Api/Models/Response/Organizations/OrganizationResponseModel.cs

```diff
@@ -81,6 +86,11 @@ public OrganizationResponseModel(Organization organization, string obj = "organi
     public bool UseCustomPermissions { get; set; }
     public bool SelfHost { get; set; }
     public bool HasPublicAndPrivateKeys { get; set; }
+    public bool UsePasswordManager { get; set; } = true;
```

Comment:
This default initialization value seems to be ignored since it's already initialized by the constructor. What's the reason for including it here? Perhaps just to document the intended default value?

Reply:
That's a good point, what do you think @cyprain-okeke?

Reply:
Cy's response

> I agree with you but The = true part of the declaration assigns a default value to the property. It means that if no value is explicitly assigned to UsePasswordManager, it will default to true.
>
> So, in this case, the property UsePasswordManager will default to true if no value is explicitly assigned to it.

Reply:
Yes but it's a non-nullable `bool`, so it will always have a value assigned in the constructor, and this default assignment will never be used. (Please correct me if I'm wrong!)